html-css/week1/yasi-portfoilo/index.html

```diff
@@ -0,0 +1,189 @@
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="shadow-effect">
+            <div class="skill">
+              <div class="skill-container">
+                <div class="skill-head">
+                  <img id="javascript" class="skill-icon" src="./resources/icons/javascript.svg" alt="JavaScript" />
+                  <h3 class="skill-heading">JavaScript</h3>
+                </div>
+                <div class="skill-body">
+                  <p>
+                    Learning JavaScript to add interactivity and dynamic functionality to web applications.
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="shadow-effect">
+            <div class="skill">
+              <div class="skill-container">
+                <div class="skill-head">
+                  <img id="design" class="skill-icon" src="resources/icons/figma-logo.svg" alt="Design Tools" />
+                  <h3 class="skill-heading">Design Tools</h3>
+                </div>
+                <div class="skill-body">
+                  <p>
+                    Basic knowledge of design tools like Figma for creating wireframes and prototypes.
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="fade-in-proj" id="projects">
+      <div class="main-content container mulish-reg">
+        <h2 class="main-heading">
+          <span class="number fira-code-reg">// 03. </span>Projects
+        </h2>
+        <div class="projects-container container"></div>
+      </div>
+    </section>
+    <section class="fade-in" id="contact">
+      <div class="main-content container mulish-reg">
+        <h2 class="main-heading">
+          <span class="number fira-code-reg">// 04. </span>Contact Me
+        </h2>
+        <form class="contact-form">
+          <div class="form-control">
+            <label for="name">Full name <span class="requirement">*</span></label>
+            <input class="form-input" type="text" id="name" name="name" placeholder="name" />
+          </div>
+          <div class="form-control">
+            <label for="email">Email address <span class="requirement">*</span></label>
+            <input class="form-input" type="email" id="email" name="email" placeholder="email" />
+          </div>
+          <div class="form-control">
+            <label for="message">Message <span class="requirement">*</span></label>
+            <textarea class="mulish-reg form-input" id="message" name="message" placeholder="message"></textarea>
+          </div>
+          <button class="form-button">Submit</button>
+        </form>
+      </div>
+    </section>
+    <footer>
+      <div class="mulish-reg footer-wrapper">
+        <p>© 2025 Yasaman Ayazi</p>
+      </div>
+    </footer>
```

Comment:
Two suggestions here. One is to remove the div and add the styling directly on the <footer>. Second is to use the <small> tag, which is made for copyright and disclaimers areas - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small
```suggestion
    <footer class="mulish-reg footer-wrapper">
        <small>© 2025 Yasaman Ayazi</small>
    </footer>
```